discord-webhook.ts

```diff
@@ -166,7 +166,8 @@ function createTargetText(
 	permRef: string | undefined,
 	repo: string,
 ) {
-	const repoText = repo !== 'vitejs/vite' ? `${repo}:` : ''
+	const repoText =
```

Comment:
should we still add a repotext to easier distinguish rolldown-vite runs or is that visible on the cards in discord already?

Reply:
Ah, that would be better. Even though it will be in a separate channel, it'll still be confusing without the repo name.